### DISCUSSION TITLE
Password could use type-twice verification or "show password" option

### DISCUSSION THREAD

**Discussion by Agit1122** [aonez/Keka#1186] [Ideas]
I've been helping a novice remotely with their Mac (I just have Win10) and was surprised that Keka's save-password option lacks the ability to view it as you type, or the common method of repeating it twice for blind verification.

Being viewed remotely, the user wanted to type the password unseen and not wonder if they erred. Of course the password could be tested with a demo archive, then used with assurance after that, but instant confirmation features are usually standard.

Thanks, if this can be added.

**Comment by aonez** [comment]
Not sure in what window you're asking that one. The main window has the verification input and the default password in the preferences has the "Show the password" option.
![Screenshot 2022-11-29 at 10 05 09](https://user-images.githubusercontent.com/480163/204485883-207fd9ab-6c28-4c48-9ce4-1183c8d0c55e.png)
![Screenshot 2022-11-29 at 10 05 54](https://user-images.githubusercontent.com/480163/204486060-030bd71c-f3a9-41c1-8a2e-56357b4cb258.png)

**Comment by Agit1122** [comment]
I figured I may have overlooked something,. but could have sworn I saw the same dialog minus those options. Doubtful that Keka just added that after we installed it 2 weeks ago! I'm not keen on how macOS freezes all menus at the top, but I thought I saw all options.

Should have waited to ask this after using it more. These are rushed remote-assists with someone who lacks patience to linger on settings.

Thanks.

**Reply by aonez** [reply]
> someone who lacks patience to linger on settings

Probably 90% of computer users 😅

**Reply by Agit1122** [reply]
I did another remote session and easily saw the type-twice password window, but there's a new mystery as to why it has no SAVE option. All you can do is minimize or close that small window.

Is that because it's a one-time per session password by design? It seems that once you close that window, the password is either hidden from view (empty fields) in the next session, or gets overwritten by typing a new one.

I'd expect to at least see "********" placeholders to indicate a saved password from before.
